src/screens/earn/farm-item/farm-item.tsx

```diff
@@ -140,43 +146,53 @@ export const FarmItem: FC<Props> = ({ farm, lastStakeRecord }) => {
   }, [lastStakeRecord?.lastStakeId, farm.item.contractAddress]);
 
   return (
-    <View style={styles.root}>
+    <View style={[styles.root, styles.mb16]}>
       <View style={styles.bageContainer}>
         {farm.item.type === PoolType.STABLESWAP && <Bage text="Stable Pool" color="#46BC94" style={styles.bage} />}
-        {new BigNumber(farm.item.vestingPeriodSeconds).isGreaterThan(SECONDS_IN_DAY) && <Bage text="Long Term" />}
+        {new BigNumber(farm.item.vestingPeriodSeconds).isGreaterThan(SECONDS_IN_DAY) && <Bage text="Long-Term Farm" />}
       </View>
       <View style={styles.mainContent}>
         <View style={[styles.tokensContainer, styles.row]}>
           <FarmTokens stakeTokens={stakeTokens} rewardToken={rewardToken} />
           <View>
             <Text style={styles.apyText}>APY: {apr}%</Text>
             <View style={styles.earnSource}>
-              <Icon name={IconNameEnum.QsEarnSource} />
+              <Icon style={styles.earnSourceIcon} name={IconNameEnum.QsEarnSource} />
               <Text style={styles.attributeTitle}>Quipuswap</Text>
             </View>
           </View>
         </View>
 
         <View style={[styles.row, styles.mb16]}>
-          <View>
+          <View style={styles.flex}>
             <Text style={styles.attributeTitle}>Your deposit:</Text>
-            <Text style={styles.attributeValue}>≈ {depositAmountAtomic.toFixed(DEFAULT_DECIMALS)}$</Text>
+            <Text style={styles.attributeValue}>≈ {depositAmountAtomic}$</Text>
```

Comment:
I would rather use `FormattedAmount` component here and provide displaying equity in the currency that was selected in app settings